convert/Makefile

```
@@ -0,0 +1,575 @@
+# convert/Makefile
+# This makefile requires GNU Make.
+
+# Version of and paths to pristine Debian Chromium source tree + tarball,
+# obtainable by running "apt-get source chromium" on a Debian system
+#
+VERSION      = 118.0.5993.70
+ORIG_SOURCE  = chromium-$(VERSION)
+ORIG_TARBALL = chromium_$(VERSION).orig.tar.xz
+
+# Uncomment this if you wish to modify $(ORIG_SOURCE), otherwise the
+# conversion process will operate on a copy.
+#
+#INPLACE = 1
+
+# Path to local ungoogled-chromium Git repo
+# https://github.com/ungoogled-software/ungoogled-chromium
+#
+UNGOOGLED = $(HOME)/src/ungoogled-chromium
```

I think based on the README this should be
```suggestion
UNGOOGLED = $(DEBIAN_CONVERT)/ungoogled-chromium
```

---

I'd like to avoid suggesting the placement of one Git repo inside another---that's kind of iffy when the juxtaposition isn't explicitly intended---but perhaps a more natural default is `$(PWD)/ungoogled-chromium`. That effectively means "clone the repo into the workspace," and if the workspace is in `DEBIAN_CONVERT`, then it'll end up the same anyway.

---

Ah yep, lets go with `PWD` then